src/Klaviyo.ts

```diff
@@ -1,5 +1,3 @@
-import { TurboModuleRegistry } from 'react-native';
```

Comment:
Renaming this file was probably the least obvious yet most important change. Evidently the `Native*` prefix is detected by codegen. 
I had expected that renaming this, in combination with the build.gradle changes above, _should have_ fixed our new arch code gen issues... but didn't. That's why I suspect there might have been an additional underlying bug in the RN/gradle libs but didn't have the time/bandwitdth to track it down myself. Besides which, this was a good time to update to latest RN version anyways so it wasn't really worth the effort.

Reply:
Neat find!